packages/api/tests/entities/Assignment.test.ts

```diff
@@ -1,16 +1,36 @@
 import { TestClient } from '../utils/TestClient';
+import { PathInput } from '../../types';
+import { ModuleType } from '../../src/Module/Module.entity';
 import * as random from '../../src/Database/seeders/random';
 
 beforeAll(async () => { await TestClient.start(); });
 afterAll(async () => { await TestClient.stop(); });
 
+export const assignment1 = "";
+
+export const moduleInput = {
```

Comment:
Couldn't this be with random module?

Reply:
I was having a hard time using the random creation for both module and path. This is why I just went ahead and created them myself. Is there an example I can use for reference?